scripts/release/build/build_info.py

```diff
@@ -3,11 +3,17 @@
 from typing import Dict
 
 from scripts.release.build.build_scenario import BuildScenario
-from scripts.release.constants import DEFAULT_REPOSITORY_PATH, DEFAULT_CHANGELOG_PATH, RELEASE_INITIAL_VERSION_ENV_VAR, \
```

Comment:
nit: Do we have diverging linter configuration ? 
I assume this change is coming from `black`.

The change in this PR looks closer to the structure we usually have